### DISCUSSION TITLE
Command for anti afk players

### DISCUSSION THREAD

**Discussion by gazdyy** [Zrips/CMI#6555] [Enhancement]
in config is already option for afk players, like kick, please add command for anti afk players

**Comment by CloudeLecaw** [comment]
U can use use the afk command for that. Its a toggle command

> cmi afk (-p:playerName) (reason) (-s)
> Toggle afk mode. Reason could be provided

Also you could create aa unafk custom alais that get executed when player is afk (check afk state) and give feadback meassage if player is not afk (check afk state)

what you mean with anti afk?
also you can check afk state and toggle it on or off depending on the afk state

So what should be the sense of an extra anti afk? may you thought of an extra /unafk command? this can be done with the afk check to if wanted

**Comment by mrfloris** [comment]
Everybody that isn't in AFK, is 'anti' or not-in-afk, someone who's AFK-ing and trying to be anti-afk, there are some exploit fixes for this in config.yml you can turn on. If you have further afk exploit fixes you want, please give specific examples and how to reproduce it
